care/facility/migrations/0487_migrate_diagnosis.py

```
@@ -0,0 +1,117 @@
+        .exclude(consultation__deleted=True)
+        .order_by("id")
+        .select_related("consultation__patient", "consultation")
+    )
+    paginator = Paginator(symptoms, 2000)
+
+    for page_number in paginator.page_range:
+        page = paginator.page(page_number)
+        bulk = []
+        for diagnosis in page.object_list:
+            if diagnosis.consultation.deleted:
+                print("consultation deleted", diagnosis.consultation.id)
```

Ruff (0.8.2)

`print` found

(T201)